jetty-start/src/main/java/org/eclipse/jetty/start/Main.java

```diff
@@ -325,12 +326,22 @@ public StartArgs processCommandLine(String[] cmdLine) throws Exception
         modules.registerAll();
 
         // 4) Active Module Resolution
-        for (String enabledModule : modules.getSortedNames(args.getEnabledModules()))
+        List<String> selectedModules = args.getEnabledModules();
```

Comment:
To reduce confusion, can we keep to one verb. Use either selected or enabled, but not both.
```suggestion
        List<String> enabledModules = args.getEnabledModules();
```

Reply:
See prior comment about what this method `StartArgs.getEnabledModules()` actually does. (also see javadoc)

Reply:
When this variable is used, it is always the list of enabled modules.   The `sortedSelectedModules` and the `unknownModules` collections are created from what should be called `enabledModules`.